src/App.js

```diff
@@ -97,8 +97,8 @@ const App = ({ locale, location, dependencies: { appSessionRef, sessionManager }
           <PrivateRoute path="/reports/" exact requireSiteTracking component={Reports} />
           <PrivateRoute path="/integrations/shopify" exact component={Shopify} />
           <PrivateRoute path="/reports/master-subscriber" exact component={MasterSubscriber} />
-          <PrivateRoute path="/reports/subscriber-history" exact component={SubscriberHistory} />
-          <PrivateRoute path="/reports/subscriber-gdpr" exact component={SubscriberGdpr} />
+          <PrivateRoute path="/subscribers/:email/:section" component={Subscribers} />
```

Comment:
We can add exact now, right?

```suggestion
          <PrivateRoute path="/subscribers/:email/:section" exact component={Subscribers} />
```